src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCosts = days * 40;
+
+  if (totalCosts >= 280) {
+    return totalCosts - 50;
+  }
+
+  if (totalCosts >= 120) {
```

Comment:
Similar to the previous comment, the discount condition should be based on the number of days rented, not `totalCosts`. Adjust the condition to reflect this.